src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,18 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC<Props> = ({ movies, query }) => (
   <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
+    {movies.map(movie => {
+      const isNeededTitle = movie.title.toLowerCase().includes(query.toLowerCase().trim()); // eslint-disable-line
+      const isNeededDesc = movie.description.toLowerCase().includes(query.toLowerCase().trim()); // eslint-disable-line
```

Comment:
You could move this logic to separate function and reuse it, because you call lots of same methods here(watch checklist pt. 2)
![image](https://user-images.githubusercontent.com/123508315/233991233-61fcfa8a-1c38-429b-a303-9f2be830cc65.png)
Also I don't think that this is the case where you can disable eslint just because you want to.